app/src/main/java/io/gnosis/safe/ui/safe/overview/SafeOverviewFragment.kt

```diff
@@ -0,0 +1,23 @@
+package io.gnosis.safe.ui.safe.overview
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import io.gnosis.safe.databinding.FragmentSafeOverviewBinding
+import io.gnosis.safe.di.components.ApplicationComponent
+import io.gnosis.safe.di.components.DaggerViewComponent
+import io.gnosis.safe.di.modules.ViewModule
+import io.gnosis.safe.ui.base.BaseFragment
+
+class SafeOverviewFragment : BaseFragment<FragmentSafeOverviewBinding>() {
+
+    override fun inject(component: ApplicationComponent) {
+        DaggerViewComponent.builder()
+            .applicationComponent(component)
+            .viewModule(ViewModule(context!!))
```

Comment:
basically all fragments will use the same viewmodule so you can actually build the `DaggerViewComponent` (or at least the builder) in the `BaseFragment` to avoid that we have to do this everywhere again